Modules/Segmentation/SuperPixel/include/itkSLICImageFilter.hxx

```diff
@@ -321,7 +321,7 @@ SLICImageFilter<TInputImage, TOutputImage, TDistancePixel>::ThreadedUpdateCluste
     itOut.NextLine();
   }
 
-  // TODO improve merge algoithm
+  // TODO improve merge algorithm
```

Comment:
someone has eagle-eyes (or a spell checker), looked at this for a couple of seconds till I spotted the missing r, and that's after I knew where I should be looking